sdk/workflow-tengo/src/pt/index.lib.tengo

```diff
@@ -447,7 +471,40 @@ workflow := func() {
 				ptablerCmdBuilder.saveFileContent(outContentFile)
 			}
 
-			return ptablerCmdBuilder.run()
+			for frameName, specsById in outFramesSpecs {
+				columnIds := []
+				for columnId, _ in specsById {
+					columnIds = append(columnIds, columnId)
+				}
+				tplArgs := {
+					frameName: frameName,
+					columnIds: columnIds
+				}
+				ptablerCmdBuilder.processWorkdir(frameName, importDirTpl, tplArgs)
```

Comment:
It is better to make one workdir process template for all frames, to reduce resulting graph complexity.